python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -844,6 +853,15 @@ def main():
             sys.stderr.write('"_scandir" missing.\n')
             exit_code = 18
 
+    # Some OS'es are not supported by upstream psutil.
+    # Windows is special. We build on XP, but XP/2003 not supported any more.
+    if not chevah_os in [ 'aix53', 'hpux1131', 'windows' ]:
```

Comment:
I think that windows should not be included here.... instead we should have a separate test for windows in which we take into consideration the windows version/

You can use this code to get the windows version in the format nt-6.1 nt-6.2

```
    if os.name == 'nt':
        parts = platform.version().split('.')
        return 'nt-%s.%s' % (parts[0], parts[1])
```

and then ignore import error only on some Windows version....

with that in mind, we need a separate python-package slave for  Windows... maybe based on Windows-2016 so that we can run this test on Windows.

Reply:
Also, a better test would be to try to import psutil and then ignore only ImportError... in this way we can cover the case in which psutil might be there but raises a different error

Reply:
Thanks for the suggestions… I've added a pointer to these comments at https://trac.chevah.com/ticket/5185#comment:1.